upstream dependency conflict

When I run `npm i` I get the next error:

```
 ✗ npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"^17.0.0" from the root project
npm ERR!   peer react@">= 16.8.4 < 18.0.0" from @nivo/calendar@0.69.0
npm ERR!   node_modules/@nivo/calendar
npm ERR!     @nivo/calendar@"^0.69.0" from the root project
npm ERR!   25 more (@nivo/core, @nivo/line, @nivo/pie, @react-hook/hover, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@">= 16.8.4 <= 17.0.0" from @nivo/colors@0.69.0
npm ERR! node_modules/@nivo/line/node_modules/@nivo/colors
npm ERR!   @nivo/colors@"0.69.0" from @nivo/line@0.69.1
npm ERR!   node_modules/@nivo/line
npm ERR!     @nivo/line@"^0.69.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/itamard/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/itamard/.npm/_logs/2021-05-20T04_14_55_630Z-debug.log
```

**Expected behavior:**
A clear installation

**Desktop:**
 - OS: Ubuntu 20.10

--- a/packages/colors/package.json
+++ b/packages/colors/package.json
@@ -37,7 +37,7 @@
   "peerDependencies": {
     "@nivo/core": "0.69.0",
     "prop-types": ">= 15.5.10 < 16.0.0",
-    "react": ">= 16.8.4 <= 17.0.0"
+    "react": ">= 16.8.4 < 18.0.0"
   },
   "publishConfig": {
     "access": "public"

--- a/packages/marimekko/package.json
+++ b/packages/marimekko/package.json
@@ -36,7 +36,7 @@
   },
   "peerDependencies": {
     "@nivo/core": "0.69.0",
-    "react": ">= 16.8.4 <= 17.0.0"
+    "react": ">= 16.8.4 < 18.0.0"
   },
   "publishConfig": {
     "access": "public"

